packages/sdk-node/src/clerkClient.ts

```diff
@@ -1,14 +1,25 @@
-import type { ClerkOptions } from '@clerk/backend';
-import { createClerkClient as _createClerkClient, verifyToken } from '@clerk/backend';
+import type { ClerkOptions, VerifyTokenOptions } from '@clerk/backend';
+import { createClerkClient as _createClerkClient, verifyToken as _verifyToken } from '@clerk/backend';
 
 import { createClerkExpressRequireAuth } from './clerkExpressRequireAuth';
 import { createClerkExpressWithAuth } from './clerkExpressWithAuth';
 import { loadApiEnv, loadClientEnv } from './utils';
 
+type MakeOptionalSecondArgument<T> = T extends (a: string, b: infer U) => infer R ? (a: string, b?: U) => R : never;
+type VerifyTokenWithOptionalSecondArgument = MakeOptionalSecondArgument<typeof _verifyToken>;
+
 type ClerkClient = ReturnType<typeof _createClerkClient> & {
   expressWithAuth: ReturnType<typeof createClerkExpressWithAuth>;
   expressRequireAuth: ReturnType<typeof createClerkExpressRequireAuth>;
-  verifyToken: typeof verifyToken;
+  verifyToken: VerifyTokenWithOptionalSecondArgument;
+};
+
+const buildVerifyToken = (params: VerifyTokenOptions) => {
+  return (...args: Parameters<VerifyTokenWithOptionalSecondArgument>) =>
+    _verifyToken(args[0], {
+      ...params,
+      ...args[1],
```

Comment:
⛔ In case of 2nd parameter missing, this line will raise the following error:
```
Uncaught TypeError: args[1] is not iterable (cannot read property undefined)
```
Could you add 2 tests (with options, without options) to verify that everything works as expected?